docs/content/dev/getting-started.md

```diff
@@ -24,22 +24,21 @@ recommend caddy and the provided configuration.
      cd hedgedoc
      ```
 4. Install the dependencies in repo root directory with `yarn install`
-5. Go to `hedgedoc/commons` directory with `cd ../commons`
-6. Build the commons package with `yarn build`
-7. Goto `hedgedoc/backend` directory with `cd ../backend`
-8. Create the `.env` config file by copying the example: `cp .env.example .env`
-9. Add a value to `HD_SESSION_SECRET` in the .env file. This can be any string, which has to be a secure password for production but can be set to simple string for debug purpose.
-10. Execute the following lines
-   ```shell
-  echo "HD_AUTH_LOCAL_ENABLE_LOGIN=true" >> .env
-  echo "HD_AUTH_LOCAL_ENABLE_REGISTER=true" >> .env
-   ```
-11. Start the backend using `yarn start:dev`
-12. Go to  `hedgedoc/frontend` directory with `cd ../frontend`
-13. Start the frontend using `yarn run dev:with-local-backend`
-14. Go to `hedgedoc/dev-reverse-proxy` with `cd ../dev-reverse-proxy`
-15. Start Caddy using `./caddy run` (if you've downloaded the binary manually) or `caddy run` (if you've installed Caddy using a package manager)
-16. Use your browser to go to <http://localhost:8080>
+5. Goto `hedgedoc/backend` directory with `cd ../backend`
```

Comment:
Why `../backend`? Following the guide step-by-step, we're still in the repo root directory.
Furthermore, we first download caddy to `hedgedoc/dev-reverse-proxy` and then create the `hedgedoc` folder afterwards by the `git clone`. This won't work.